docs/prompt_tuning/manual_prompt_tuning.md

```diff
@@ -6,49 +6,85 @@ We provide a means for you to do this by allowing you to specify a custom prompt
+[Prompt Source](http://github.com/microsoft/graphrag/blob/main/graphrag/prompts/index/claim_extraction.py)
 
-### Tokens (values provided by extractor)
+#### Tokens
 
 - **{input_text}** - The input text to be processed.
 - **{tuple_delimiter}** - A delimiter for separating values within a tuple. A single tuple is used to represent an individual entity or relationship.
 - **{record_delimiter}** - A delimiter for separating tuple instances.
 - **{completion_delimiter}** - An indicator for when generation is complete.
-
-Note: there is additional parameter for the `Claim Description` that is used in claim extraction.
-The default value is
-
-`"Any claims or facts that could be relevant to information discovery."`
+- **{entity_specs}** - A list of entity types.
+- **{claim_description}** - Description of what claims should look like. Default is: `"Any claims or facts that could be relevant to information discovery."`
 
 See the [configuration documentation](../config/overview.md) for details on how to change this.
 
-## Generate Community Reports
+### Generate Community Reports
 
-[Prompt Source](http://github.com/microsoft/graphrag/blob/main/graphrag/prompts/community_report.py)
+[Prompt Source](http://github.com/microsoft/graphrag/blob/main/graphrag/prompts/index/community_report.py)
 
-### Tokens (values provided by extractor)
+#### Tokens
 
 - **{input_text}** - The input text to generate the report with. This will contain tables of entities and relationships.
+
+## Query Prompts
+
+### Local Search
+
+[Prompt Source](http://github.com/microsoft/graphrag/blob/main/graphrag/prompts/query/local_search_system_prompt.py)
+
+#### Tokens
+
+- **{response_type}** - Describe how the response should look. We default to "multiple paragraphs".
+- **{context_data}** - The data tables from GraphRAG's index.
+
+### Global Search
+
+[Mapper Prompt Source](http://github.com/microsoft/graphrag/blob/main/graphrag/prompts/query/global_search_map_system_prompt.py)
+
+[Reducer Prompt Source](http://github.com/microsoft/graphrag/blob/main/graphrag/prompts/query/global_search_reduce_system_prompt.py)
+
+[Knowledge Prompt Source](http://github.com/microsoft/graphrag/blob/main/graphrag/prompts/query/global_search_knowledge_system_prompt.py)
+
+Global search uses a map/reduce approach to summarization. You can tune these prompts independently. This search also includes the ability to adjust the use of general knowledge from the model's training.
+
+#### Tokens
+
+- **{response_type}** - Describe how the response should look (reducer only). We default to "multiple paragraphs".
+- **{context_data}** - The data tables from GraphRAG's index.
+
+### Drift Search
+
+[Prompt Source](http://github.com/microsoft/graphrag/blob/main/graphrag/prompts/query/drift_search_system_prompt.py)
+
+#### Tokens
+
+- **{response_type}** - Describe how the response should look. We default to "multiple paragraphs".
```

Comment:
Right now Drift does not support response type, at least until I add the reduce prompt

Reply:
The token is in the prompt though, so if someone is manually tuning they can write in what they want here.